lib/beacon-verification.ts

```diff
@@ -1,6 +1,8 @@
-import { bls12_381 as bls } from '@noble/curves/bls12-381'
-import { sha256 } from '@noble/hashes/sha256'
-import { ensureBytes }  from '@noble/curves/abstract/utils'
+import {bls12_381 as bls} from '@noble/curves/bls12-381'
+import {bn254} from '@kevincharm/noble-bn254-drand'
```

Comment:
Can't we use noble directly?

Reply:
I think this is necessary to get the right hash to curve